Demo/src/main/java/com/braintreepayments/demo/DemoActivity.java

```diff
@@ -49,7 +50,13 @@ public BraintreeClient getBraintreeClient() {
         if (braintreeClient == null) {
             if (Settings.useTokenizationKey(this)) {
                 String tokenizationKey = Settings.getTokenizationKey(this);
-                braintreeClient = new BraintreeClient(this, tokenizationKey);
+
+                boolean useAppLink = Settings.getPayPalLinkType(this).equals(getString(R.string.paypal_app_link));
+                Uri appLinkUri = null;
+                if (useAppLink) {
+                    appLinkUri = Uri.parse("https://example.com");
```

Comment:
Can we use the actual site @scannillo set up here or is it not ready yet: "https://mobile-sdk-demo-site-838cead5d3ab.herokuapp.com/"?

Reply:
Ah, I wasn't aware. I'll update this.